oak-core/src/test/java/org/apache/jackrabbit/oak/security/authentication/user/LoginModuleImplTest.java

```diff
@@ -106,7 +105,6 @@ public class LoginModuleImplTest extends AbstractSecurityTest {
     @Override
     public void after() throws Exception {
         try {
-            clearInvocations(monitor);
```

Comment:
i checked in my local oak and there are more usages of 'Mockito.clearInvocations()' in subclasses of the AbstractSecurityTest both in oak-core and in other security modules.

i am curious to understand why you remove this one and not the others.

Reply:
With this line the test failed with:
```
org.mockito.exceptions.misusing.NotAMockException: Argument should be a mock, but is: class org.apache.jackrabbit.oak.spi.security.authentication.LoginModuleMonitor$MockitoMock$AIxpL0Xb
	at org.apache.jackrabbit.oak.security.authentication.user.LoginModuleImplTest.after(LoginModuleImplTest.java:110)
```
It seems the sequence how multiple After methods on the same class are called is determined by method name (lexicographic ordering?). It means the existing `after()` method is called first and then the newly introduced `clearMocks()`.

I will revert the change in `LoginModuleImplTest` and move `Mockito.framework().clearInlineMocks()` to the existing `after()` method. The changes to `LoginContextProviderImplTest` will still be necessary though.